src/App.js

```diff
@@ -6,18 +6,30 @@ import PersonDetails from "./components/PersonDetails"
 
 import { data } from "./store/data"
 
+
+// const person =  [{
+//   id: 1,
+//   name: "Creola Katherine Johnson", 
+//   profession: "mathematician",
+//   accomplishment: "spaceflight calculations",
+//   imageUrl: "https://i.imgur.com/yXOvdOSs.jpg"
+// }]
+
 function App() {
 
   return (
     <div className="App">
       <header><h1>Scientists</h1></header>
       <section>
         {data.map((person) => (
+          <li key={person.id}>
```

Comment:
Good work with adding `key` attributes 🌸
Also wrapping these Cards inside `<li>` elements is better approach for accessibility and search engine optimization (but you should always use `<ul>` or `<ol>` elements as parent)

```
<section>
    <ul>
        {data.map((person) => (
          <li key={person.id}> ...